modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -227,53 +226,55 @@ rpk topic alter-config <topic> --set segment.ms=<segment_ms_duration>
 [[space_management]]
 == Space management
 
-Redpanda divides disk storage into different categories to provide a flexible configuration of space:
+NOTE: Space management only works when xref:../tiered-storage.adoc[Tiered Storage] is enabled on all topics. Space management and the housekeeping process only considers removing data that is safely stored in Tiered Storage.
+
+The goal of space management is to utilize the local disk space as a cache. It works alongside xref:../tiered-storage.adoc[Tiered Storage] to provide faster access to recent data, while making sure that local disk space is managed in accordance with retention policies and other processes, such as xref:./continuous-data-balancing.adoc[Continuous Data Balancing] and xref:./decommission-brokers.adoc[decommissioning]. 
 
-* Reserved disk space (`disk_reservation_percent`): This overhead reservation is disk space that Redpanda does not use.
- ** As disk space used by cache storage and log storage expand to their target sizes, this provides buffer space to avoid free disk space alerts.
- ** Because SSDs that run near capacity can experience performance degradation, this provides buffer space to prevent running a device at capacity.
-* Cache storage (`cloud_storage_cache_size_percent` or `cloud_storage_cache_size`): This is the maximum size of the xref:manage:tiered-storage.adoc#caching[disk cache] used if Tiered Storage is enabled. As the cache reaches its limit, new data added to the cache removes old data from the cache.
-* Log storage (`retention_local_target_capacity_percent` or `retention_local_target_capacity_bytes`): This log data reservation is the disk space used as the target maximum size for user data, as well as Redpanda internal topics, like the control log. It's generally about 70-80% of total disk space.
+Space management divides the disk space into different areas that can be managed separately:
+
+* Reserved disk space (`disk_reservation_percent`) is the area of disk space that Redpanda does not use.
+** As the disk space used by the other areas grows to their target sizes, this provides a buffer space to avoid free disk space alerts.
```

Comment:
```suggestion
** As the disk space used by the other areas grows to their target sizes, the reserved space provides buffer space to avoid free disk space alerts.
```